app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt

```diff
@@ -82,7 +81,7 @@ class ProfileFragment : Fragment() {
     }
 
     override fun onOptionsItemSelected(item: MenuItem?): Boolean {
-        when (item?.getItemId()) {
+        when (item?.itemId) {
```

Comment:
Why this change?

Reply:
> Why this change?

It automatically converts. But after check commit different when I try to edit it, studio automatic input itemId instead of getItemId(). Should I recover it?

Reply:
OK, let it be